### ISSUE TITLE
Batch Normalization Exercises: strides logic in conv_layer() inconsistent with the writeup

### ISSUE THREAD

**Issue by agoila** [udacity/dlnd-issue-reports#369]
Refer notebook here: https://github.com/udacity/deep-learning/blob/master/batch-norm/Batch_Normalization_Exercises.ipynb

The writeup for cell 3 says that we use strides of 1x1 on layers with odd depths, and strides of 2x2 on layers with even depths. However, the code implementation uses the reverse logic.

**Comment by mcleonard** [comment]
Thanks! Will fix this.

Internal tracking: DLND-1162
